tests/test_cli.py

```diff
@@ -51,7 +58,7 @@ def super_long_function_with_long_params(
     pass
 
 
-class LoudContextConfig(DefaultConfig):
+class LoadContextConfig(DefaultConfig):
```

Comment:
This was not a typo, it's a "LoudContextConfig" because it prints "loudly" when running. Please revert the change :)

Reply:
Opps! Sure.

Reply:
Done.